Godeps/_workspace/src/k8s.io/kubernetes/pkg/apis/extensions/install/install.go

```diff
@@ -33,6 +33,8 @@ import (
 	"k8s.io/kubernetes/pkg/apis/extensions/v1beta1"
 	"k8s.io/kubernetes/pkg/runtime"
 	"k8s.io/kubernetes/pkg/util/sets"
+
+	_ "k8s.io/kubernetes/pkg/apis/batch/install"
```

Comment:
@smarterclayton It wasn't just this (or the other way around)?  Both go 1.4 and 1.6 are happy for me locally.